src/pybamm/experiment/step/base_step.py

```diff
@@ -74,7 +76,7 @@ def __init__(
         self.input_duration = duration
         self.input_value = value
         # Check if drive cycle
-        is_drive_cycle = isinstance(value, np.ndarray)
+        is_drive_cycle = isinstance(value, npt.NDArray)
```

Comment:
This should not be changed as we are checking if `value` is an instance of `np.ndarray`. We could have changed it if we were checking the type of the `value`, but we don't want that.